src/less/z-index.less

```diff
@@ -3,42 +3,53 @@
 /* z-index issues.
 /*****************************/
 
+@zIndex_0: 0;
```

Comment:
i know our naming is all over the place now, but since we're converging on camel case in all other languages, we might as well do the same with our less - `@zIndex0`, etc in this case